saleor/app/types.py

```diff
@@ -0,0 +1,5 @@
+class AppType:
+    CUSTOM = "custom"
+    EXTERNAL = "external"
```

Comment:
Wondering if this will be clear enough but I don't have any better name :/

Reply:
If we don't have better names we can at least add a comment here. For example: 
Custom - App integrated through webhooks. 
External - App integrated through the python interface.

Reply:
Agree with Owczar.

Reply:
I don't think these fit: `CUSTOM` is really "created by the staff" and `EXTERNAL` is "installed using a manifest".

Reply:
@patrys Any ideas for enum names? It's important to sort out naming issues early, as this later escalates to the entire codebase (plugin vs. extension vs. app).

Reply:
Maybe `LOCAL` and `THIRDPARTY`?